### PR TITLE
(fixes #188) Update documentation for Firefox searchplugins

### LINKED ISSUE #188: Update documentation for Firefox searchplugins
Firefox moved to a completely different system, need to remove most of it and point to existing docs in-tree.

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: src/products/searchplugins/setup_searchplugins.md [hunk 1]
```diff
@@ -1,193 +1,119 @@
...
+The *Name* to use in your localization is `ویکیپیڈیا (ur)`. Copying and pasting RTL languages is particularly tricky.
 
 ## Setting up files for locale’s repository
 
 ### region.properties
 
-region.properties is stored in `/browser/chrome/browser-region` for Firefox desktop (`/mobile/chrome` for Firefox for Android), and it contains information about protocol handlers. You can use these files as a base:
-* For Firefox desktop [this region.properties model](files/desktop_region.properties).
-* For Firefox for Android [this region.properties model](files/mobile_region.properties).
+region.properties is stored in `/browser/chrome/browser-region` for Firefox desktop, and it contains information about protocol handlers. You can use [this region.properties model](files/desktop_region.properties) as a base .
```
Comment (Delphine) [inline src/products/searchplugins/setup_searchplugins.md]: Extra space before the period at the end.

**Review by Delphine — CHANGES_REQUESTED** [review]
I spotted just one nit. Otherwise, this LGTM

File: src/products/searchplugins/setup_searchplugins.md (hunk 1 above)
Comment (flodolo) [reply src/products/searchplugins/setup_searchplugins.md]: Thanks, fixed 👍

**Review by Delphine — APPROVED** [review]
LGTM now!

**Pull request merged**